index.html

```diff
@@ -25,7 +26,121 @@ <h1 class="header2">I'm Ali<br>Glad to see<br>you!</h1>
                 <li class="github"><a href="https://"><img src="img/Icon -GitHub.png" alt="GitHub"></a></li>
                 <li class="angelist"><a href="https://"><img src="img/Icon - Angelist.png" alt="Angelist"></a></li>
             </ul>
-        <section>
+        <section class="tonic_border">
+        <section class="tonic_project">
+            <a class="tonic_pic" href="https://"><img src="img/tonic_pic.png" alt=""></a>
+            <p class="tonic_name">Tonic</p>
```

Comment:
- [ ] This is a heading. please give it an `hx` tag. don't give h1 because its not the main heading. any where from `h2,3,...` will do. 😄